src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -430,6 +417,15 @@ private void addScmDetails(List<Fact> factsList) {
         }
     }
 
+    private Collection<? extends ChangeLogSet.AffectedFile> getAffectedFiles(ChangeLogSet.Entry entry) {
```

Comment:
Even better, you added tests 🤖

Reply:
I'm going to add tests to most of the code I refactor. Once I decide refactoring of given method is done I add tests.